<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimeSeriesControlGroupBox</class>
 <widget class="QGroupBox" name="TimeSeriesControlGroupBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>341</width>
    <height>87</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>341</width>
    <height>87</height>
   </size>
  </property>
  <property name="title">
   <string>Time-varying series control</string>
  </property>
  <layout class="QGridLayout" name="gridLayoutSelectedDatasetTMSeries">
   <item row="2" column="6" rowspan="3">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Interval (ms):</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" rowspan="2" colspan="6">
    <widget class="QSlider" name="horizontalSliderSelectedDatasetStep">
     <property name="maximum">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="2">
    <widget class="QLabel" name="labelSelectedDatasetStep">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Step:</string>
     </property>
     <property name="buddy">
      <cstring>spinBoxSelectedDatasetStep</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="5" rowspan="3">
    <widget class="QSpinBox" name="spinBoxTMIncrement">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="2" column="4" rowspan="3">
    <widget class="QLabel" name="label_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Step increment:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="7" rowspan="3">
    <widget class="QSpinBox" name="spinBoxTMInterval">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="2" column="0" rowspan="3" colspan="4" alignment="Qt::AlignLeft">
    <widget class="QWidget" name="widgetSelectedDatasetTMButtons" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="toolButtonPlay">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/icons/icons/images/icons_17.png</normaloff>
          <normalon>:/icons/icons/images/icons_19.png</normalon>:/icons/icons/images/icons_17.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButtonStart">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/icons/icons/images/icons_18.png</normaloff>:/icons/icons/images/icons_18.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButtonEnd">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/icons/icons/images/icons_20.png</normaloff>:/icons/icons/images/icons_20.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="arrowType">
         <enum>Qt::NoArrow</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" rowspan="2">
    <widget class="QSpinBox" name="spinBoxSelectedDatasetStep">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximum">
      <number>0</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>spinBoxSelectedDatasetStep</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSliderSelectedDatasetStep</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>61</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>92</x>
     <y>34</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderSelectedDatasetStep</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxSelectedDatasetStep</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>173</x>
     <y>37</y>
    </hint>
    <hint type="destinationlabel">
     <x>66</x>
     <y>37</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
